src/lib.rs

```diff
@@ -57,6 +61,21 @@ impl Retry {
         self
     }
 
+    /// Enable jitter.
+    pub const fn jitter(mut self, enabled: bool) -> Self {
+        self.enable_jitter = enabled;
+        self
+    }
+
+    fn apply_jitter(&self, delay: Duration) -> Duration {
+        if self.enable_jitter {
+            // [0.5, 1.0)
+            delay.mul_f64(0.5 + rand::random::<f64>() / 2.0)
+        } else {
+            delay
+        }
+    }
```

Comment:
Nice, it seems consistent with industry standards https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/core/retry/backoff/EqualJitterBackoffStrategy.html